lib/hashie/extensions/persistable.rb

```diff
@@ -0,0 +1,38 @@
+  module Hashie
+    module Extensions
+      module Persistable
+        CANNOT_INCLUDE = 'Peristable can only be used on classes with Hashie::Extensions::HashInitializer'
```

Comment:
Got it.

I think the message is important because a user probably won't know what a MissingHashInitializer means. I can see the value of a localizable system, but I'd rather have that be a separate issue/PR, since there's other errors with messages (e.g. [here](https://github.com/intridea/hashie/blob/master/lib/hashie/extensions/deep_fetch.rb#L23) and [here](https://github.com/intridea/hashie/blob/a18864fe1d079daf3eda5cf14a168ef6abd6403c/lib/hashie/extensions/coercion.rb#L46)).

How about if I do this, and then you can create a separate issue for localization of error messages?

```ruby
class MissingHashInitializer < StandardError
  def initialize(msg = 'Peristable can only be used on classes with Hashie::Extensions::HashInitializer', *args)
    super
  end
end
```